src/main/resources/com/cloudbees/hudson/plugins/folder/buildable/DisableAction/index.jelly

```diff
@@ -0,0 +1,22 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <l:layout title="${%Disable.title(it.item.getDisplayName())}">
+    <st:include page="sidepanel.jelly" it="${it.item}"/>
+    <l:main-panel>
+      <h1>
+        ${%Disable.title(it.item.getDisplayName())}
+      </h1>
+
+      <form action="disable" method="POST">
+      ${%confirmationMessage}
+        <ul>
+          <j:forEach var="it" items="${it.getJobs()}">
```

Comment:
the `getAllJobs()` method on a folder return all jobs, even in nested folders. Using the same source as the action shows only the jobs that will be changed.

I agree about the CCE. I'm looking on how to give the user the list of missed jobs.